src/components/FormStep.js

```diff
@@ -296,6 +294,23 @@ const FormStep = ({
     history.push(navigateTo);
   };
 
+  const onFormIOInitialized = () => {
```

Comment:
In relation to one of the low-prio issues from sprint 9 (https://github.com/open-formulieren/open-forms/issues/974 => Logic not working with checkboxes). 

I think it is due to the fact that on initialisation there is no logic check done (so, for example, logic rules checking if a checkbox is unchecked don't run until something else is changed in the form). So maybe there should also be a check logic here in the `onFormIOInitialised`?

Reply:
Hrm, but the `onFormIOInitialised` is triggered in a re-render because of the logic check too :thinking:

Reply:
I've created open-formulieren/open-forms#994 for this, let's tackle that separately